### PR TITLE
ALBS-139: The service should be able to return only a private mirror for suitable clients

### PR DESCRIPTION
**Pull request by soksanichenko**
- Full description for format of a mirror's config

### CODE REVIEW

File: docs/Mirrors.md [hunk 1]
```diff
@@ -41,7 +41,41 @@ You can create a public AlmaLinux mirror in 6 easy steps:
    your mirror to the `mirrors.d` directory.
    You can use the [official AlmaLinux repo file](https://github.com/AlmaLinux/mirrors/blob/master/mirrors.d/repo.almalinux.org.yml)
    as an example. Your mirror does not have to provide all the protocols
-   that our main mirror provides, but either HTTP or HTTPS is required.
+   that our main mirror provides, but either HTTP or HTTPS is required. Format of a mirror's config is described below. 
+   Also, you can validate your config use some JSON online validator using [that JSON schema](https://github.com/AlmaLinux/mirrors/blob/yaml_snippets/json_schemas/mirror_config.json) and converting your config to JSON.
```
Comment (rpisarev) [inline docs/Mirrors.md:45]: "Also, you can validate your config to use some JSON online validator" more clear as for me

File: docs/Mirrors.md [hunk 2]
```diff
@@ -41,7 +41,41 @@ You can create a public AlmaLinux mirror in 6 easy steps:
...
    You can use the [official AlmaLinux repo file](https://github.com/AlmaLinux/mirrors/blob/master/mirrors.d/repo.almalinux.org.yml)
    as an example. Your mirror does not have to provide all the protocols
-   that our main mirror provides, but either HTTP or HTTPS is required.
+   that our main mirror provides, but either HTTP or HTTPS is required. Format of a mirror's config is described below. 
+   Also, you can validate your config use some JSON online validator using [that JSON schema](https://github.com/AlmaLinux/mirrors/blob/yaml_snippets/json_schemas/mirror_config.json) and converting your config to JSON.  
+    
+    ```YAML
+    ---
+    name: <domain name of a mirror, e.g. almalinux.mirror.link>
+    cloud_type: <azure|aws. It's required if a mirror is located in AWS/Azure cloud>
```
Comment (rpisarev) [inline docs/Mirrors.md:50]: "It'll be required if a mirror is located in AWS/Azure cloud" a little bit more English

File: docs/Mirrors.md [hunk 3]
```diff
@@ -41,7 +41,41 @@ You can create a public AlmaLinux mirror in 6 easy steps:
...
+    sponsor_url: <URL of a mirror's sponsor/holder, e.g. http://some.company.name>
+    email: <e-mail of a mirror's sponsor/holder, e.g. admin@some.company.name>
+    geolocation:
+      continent: <name of a continent, e.g. North America; than field is not mandatory>
+      country: <two-letter name of a country, e.g. US>
+      state_province: <name of a province, e.g. New York>
+      city: <name of a city, e.g. New York>
+    private: <true|false. A mirror is available in some local network if the param is true>
+    monopoly: <true|false. The mirrors list contains only this private mirror for a suitable client if param is true>
+    # That list is required if a mirror is local or cloud.
```
Comment (rpisarev) [inline docs/Mirrors.md:67]: "The list will be required if a mirror is local or cloud" a little bit more English style

File: docs/Mirrors.md [hunk 4]
```diff
@@ -41,7 +41,41 @@ You can create a public AlmaLinux mirror in 6 easy steps:
...
+    # It contains subnets behind which located a private mirror.
+    # Also, it can be string and contains URL which returns json list with subnets
+    subnets:
+      - <network mask>
+    # That parameter can specify ASN which is maintained by a mirror is located in cloud
+    asn: <ASN number, see https://en.wikipedia.org/wiki/Autonomous_system_(Internet)>
+    # that list is required if a mirror is located in Azure/AWS cloud
```
Comment (rpisarev) [inline docs/Mirrors.md:74]: "The list will be required if a mirror is located in Azure/AWS cloud" a little bit more English style

File: docs/Mirrors.md [hunk 5]
```diff
@@ -41,7 +41,41 @@ You can create a public AlmaLinux mirror in 6 easy steps:
...
+    cloud_regions:
+      - <cloud_region of Azure/AWS, e.g. australiacentral2>
+    ...
+   ```
 
    More information about creating a pull request can be found in the
```
Comment (rpisarev) [inline docs/Mirrors.md:80]: "You could find more information about creating a pull request in..." a little bit more English style

**Review by anfimovdm — APPROVED** [review]

**Review by Korulag — APPROVED** [review]

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**